client/config.xml

```diff
@@ -79,7 +83,6 @@
         <preference name="StatusBarBackgroundColor" value="#263238" />
         <preference name="StatusBarStyle" value="lightcontent" />
 
-        <!-- https://github.com/apache/cordova-plugin-splashscreen#launch-storyboard-images -->
```

Comment:
LInk to https://cordova.apache.org/docs/en/latest/core/features/splashscreen/index.html#ios-specific-information instead